js/checklist.js

```diff
@@ -0,0 +1,165 @@
+	 * using the Canvas API. Ideally, these would be provided to the JS as environment
+	 * variables, but since we don't have the ability to modify the server-side controller,
+	 * that's not an option.
+	 *
+	 * Here's an easy way to get the list of external tools if you know the account ID. Just
+	 * run this code from the course home page, and then inspect the objects to find the "id"
+	 * of the tool you want:
+	 * 
+	 * $.getJSON("/api/v1/accounts/39/external_tools", $.proxy(console.log, console));
+	 *
+	 * TECHNICAL RISKS:
+	 *
+	 * 1) Instructure does not support this kind of modification to the checklist, and Instructure
+	 * could release a breaking change to the CourseWizard code at any time.
+	 *
+	 * 2) The external tool links could break if this code is executed from a different account, 
+	 * or if the external tools themselves are modified such that the IDs are no longer valid.
+	 * 
+	 */
+
+	// Hard-coded external tool IDs
+	var POLICY_WIZARD_TOOL_ID = 1509; // Tool ID for account_id=39 
+	var MANAGE_PEOPLE_TOOL_ID = 3958; // Tool ID for account_id=39
+	
+	// Background image
+	// Source: http://hpac.harvard.edu/pages/photos
+	//var BACKGROUND_IMG_URL = "//hpac.harvard.edu/files/hpac/files/022210_stock_jc_047_124407_978454_1.jpg"; 
+	var BACKGROUND_IMG_URL = "//harvard-atg.github.io/canvas-course-checklist/img/john_harvard_statue.jpg";
+
+	// Base course URL (i.e. /courses/1234)
+	var BASE_COURSE_URL = window.location.pathname;
+	var DEBUG = false; // (window.location.pathname == "/courses/39");
+
+	//----- CHANGE #1 -----
+	// REMOVE: Modify "Import Content" item
+	ListItems[0].text = "If you've been using another course management system, you probably have stuff in there that you're going to want moved over to Canvas. We can walk you through the process of easily migrating your content into Canvas. If you would like to incorporate content from a previous iSite, please contact the Academic Technology Group at atg@fas.harvard.edu";
+	
+	//----- CHANGE #2 -----
+	// REMOVE: "Add Students" item
+	ListItems.splice(2, 1);
+	
+	//----- CHANGE #3 -----
+	// CHANGE: "Add TAs" item text and move up near the top of the list
+	var add_tas = ListItems.splice(6, 1)[0];
+	ListItems.splice(1, 0, add_tas);
+	$.each(['text', 'title'], function(idx, prop) {
+		add_tas[prop] = add_tas[prop].replace(/TA(s)?/g, "TF$1");
+	});
+	add_tas.url = BASE_COURSE_URL + "/external_tools/" + MANAGE_PEOPLE_TOOL_ID;
+	
+
+	//----- CHANGE #4 -----
+	// INSERT: Academic Integrity Policy tool
+	ListItems.splice(7, 0, {
+		key:'policy_wizard',
+		complete: false,
+		title: "Customize academic integrity policy",
+		text: "Customize the academic integrity policy for your course.",
+		url: BASE_COURSE_URL + "/external_tools/" + POLICY_WIZARD_TOOL_ID,
+		iconClass: 'icon-educators'
+	});
+	
+	//----- CHANGE #5 -----
+	// Modify background image.
+	if (BACKGROUND_IMG_URL) {
+		$("head").append([
+			'<style>',
+			'.ic-wizard-box {',
```

Comment:
If this CSS bit was supplied in a separate .css file, then the background image URL could just be a relative (to the CSS location) path, right? That'd make deployment easier/safer.  Are there drawbacks to that approach?